packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -162,8 +162,8 @@ export abstract class ProjectBase extends cdk.Construct implements IProject {
   /** The human-visible name of this Project. */
   public abstract readonly projectName: string;
 
-  /** The IAM service Role of this Project. Undefined for imported Projects. */
-  public abstract readonly role?: iam.IRole;
+  /** The IAM service Role of this Project. */
```

Comment:
Shouldn't `IProject` now extend `IGrantable`?

Reply:
Yeah it does in latest version.